src/custom-surf/pages/ServiceTicketDetail.tsx

```diff
@@ -65,13 +65,26 @@ interface Action {
     requiredStates?: ServiceTicketProcessState[];
 }
 
+const getUnfinishedInformLog = (ticket: ServiceTicketWithDetails): ServiceTicketLog | undefined => {
+    const unfinished = ticket.logs.filter((log) => log.completed === false);
```

Comment:
Just curious if there is a reason we don't use libraries like lodash (which I thought was pretty standard).

So you get:

return _.find(ticket.logs, log => !log.completed)

Reply:
The reason is a lack of JS experience from my side :) Nice idea, will apply it :+1